roles/openshift_logging/tasks/install_elasticsearch.yaml

```diff
@@ -101,9 +114,8 @@
     openshift_logging_es_recover_after_time: "{{openshift_logging_es_ops_recover_after_time}}"
     es_node_selector: "{{openshift_logging_es_ops_nodeselector | default({})}}"
   with_indexed_items:
-    - "{{es_dc_pool_ops | default([])}}"
+    - "{{ es_ops_dc_pool | default([]) }}"
```

Comment:
Default to empty here because ops cluster is optional but not in non-ops cluster because it's required?